script.js

```diff
@@ -0,0 +1,12 @@
+const hamburger = document.querySelector(".hamburger");
+const navMenu = document.querySelector(".nav-menu");
+
+hamburger.addEventListener("click", () => {
```

Comment:
- [ ] Kindly style your mobile menu nicely. Currently it has an overflow on the x-axis causing the `close` button to not show properly upon load.

Your design
![image](https://user-images.githubusercontent.com/81932478/195211868-a9c5ef36-b222-4da1-9012-38a57fb2b93e.png)

Given design
![image](https://user-images.githubusercontent.com/81932478/195212156-223d6070-1262-42ec-a04d-22d08c34081c.png)